### ISSUE TITLE
Prototype_schema.jpg Error?

### ISSUE THREAD

**Issue by creoguy** [mkeller0815/MEEPROMMER#9]
The pin labels for the Arduino Nano appear to be incorrect for A0-A7. On the schematic, it shows A0 as pin 11 and A7 as pin 4. Pin 4 should be A0 and Pin 11 should be A7 with A1-A6 swapped in-between as well.

**Comment by mkeller0815** [comment]
You are right. I never used or checked the pin numbers on this part. It's from a library I imported to the layout tool. It seems that the order of A0 to A7 was reversed. 
I fixed it in the picture (by just adjusting the pin numbers). It's a quickhack, but should do the job.
Thanks for taking care,
Mario.
